client/landing/stepper/declarative-flow/internals/steps-repository/plans/plans-wrapper.tsx

```diff
@@ -126,6 +140,8 @@ const PlansWrapper: React.FC< Props > = ( props ) => {
 					isInVerticalScrollingPlansExperiment={ isInVerticalScrollingPlansExperiment }
 					shouldShowPlansFeatureComparison={ isDesktop } // Show feature comparison layout in signup flow and desktop resolutions
 					isReskinned={ isReskinned }
+					hidePlansFeatureComparison={ hidePlansFeatureComparison }
```

Comment:
This works but I feel this is a bit of overkill. From the current use case, there wouldn't be a case we need a dynamic state like this. Plus, it adds up the dependency between the domains step and the plans step, since the action that set this flag is dispatched in the domains step. 

Could we simply pass a `true` here?

Reply:
I agree, but FWIW the less "prop drilling" we introduce the better, in general. Querying a store or calling a hook is far better than passing things as props. You can think of hooks (which may query stores) as globals that are acceptable.

Reply:
^ I was just merely looking for the opportunity to say this, cause we've been steering things on occasion towards prop usage (pass a prop for this or that). and similarly, I'm equally looking for the opportunity to "defeat" some recommendations for inheritance (creating subclasses) I've read at a few places 🤣

Reply:
After adding the dispatch in the domains step, I changed to have it as a [side effect of the flow](https://github.com/Automattic/wp-calypso/blob/update/newsletter-plans-copy/client/landing/stepper/declarative-flow/newsletter.ts#L53-L58). So it is no more dependant on the domains step :)

Reply:
` I was just merely looking for the opportunity to say this` hehe

Reply:
Cool. I have no objection keeping it this way then especially now that it's set at the flow level :)